code/modules/lighting/lighting_object.dm

```diff
@@ -93,7 +93,7 @@
 		affected_turf.underlays += current_underlay
 	else if(!set_luminosity)
 		affected_turf.underlays -= current_underlay
-		current_underlay.icon_state = "dark"
+		current_underlay.icon_state = "lighting_dark"
```

Comment:
could you also change the "transparent" icon_state to "lighting_transparent" so that if a turf uses that icon_state we dont have to do this again?

Reply:
Can do